app/models/concerns/post.rb

```diff
@@ -0,0 +1,61 @@
+module Post
+  extend ActiveSupport::Concern
+
+  include NameFromTitle
+  include Slug
```

Comment:
My only small issue with this is the naming. It feels inconsistent.

Feels like it should be either:
- `Name` and `Slug` or
- `NameFromTitle` (though technically it's from title and subtitle) and `SlugX` (not sure what X would be though)

Is there a pattern or naming convention for things like this?